service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateBigQueryDatasetStep.java

```diff
@@ -120,7 +121,7 @@ public StepResult doStep(FlightContext flightContext)
     // have a cleaner way of deserializing parameterized types, so we suppress warnings here.
```

Comment:
This comment needs to be fixed up (the `TypeReference<T>` is the "cleaner way").  Not sure about the `@SuppressWarnings`, overload can still return `null`.